src/Illuminate/View/Mix/Mix.php

```diff
@@ -0,0 +1,184 @@
+<?php
+
+namespace Illuminate\View\Mix;
+
+use Illuminate\Support\HtmlString;
+
+class Mix
+{
+    protected $path;
+    protected $manifest;
+    protected $manifestDirectory;
+    protected $disabled = false;
+
+    /**
+     * Get the path to a versioned Mix file or a simple message if mix is disabled.
+     *
+     * @param  string  $path
+     * @param  string  $manifestDirectory
+     * @return \Illuminate\Support\HtmlString
+     *
+     * @throws \Illuminate\View\Mix\MixException
+     */
+    public function mix($path, $manifestDirectory = '')
+    {
+        if ($this->disabled) {
+            return $this->disabledPath();
+        }
+
+        return $this->getRealPath($path, $manifestDirectory);
+    }
+
+    /**
+     * Get the path to a versioned Mix file.
+     *
+     * @param  string  $path
+     * @param  string  $manifestDirectory
+     * @return \Illuminate\Support\HtmlString
+     *
+     * @throws \Illuminate\View\Mix\MixException
+     */
+    protected function getRealPath($path, $manifestDirectory)
+    {
+        $this->init($path, $manifestDirectory);
+
+        if ($this->hmrModeEnabled()) {
+            return $this->hmrPath();
+        }
+
+        return $this->compiledPath();
+    }
+
+    /**
+     * Set a sanitized version of assets.
+     *
+     * @param  string  $path
+     * @param  string  $manifestDirectory
+     * @return void
+     */
+    protected function init($path, $manifestDirectory)
+    {
+        $this->path = $this->sanitize($path);
+        $this->manifestDirectory = $this->sanitize($manifestDirectory);
+    }
+
+    /**
+     * Get a sanitized version of a path.
+     *
+     * @param  string  $path
+     * @return string
+     */
+    protected function sanitize($path)
+    {
+        if ($path && ! starts_with($path, '/')) {
+            $path = "/{$path}";
+        }
+
+        return $path;
+    }
+
+    /**
+     * Check if the HRM mode of Mix is enabled.
+     *
+     * @return bool
+     */
+    protected function hmrModeEnabled()
+    {
+        return file_exists(public_path($this->manifestDirectory.'/hot'));
+    }
+
+    /**
+     * Get the full path to the file through the HMR server.
+     *
+     * @return \Illuminate\Support\HtmlString
+     */
+    protected function hmrPath()
+    {
+        return new HtmlString("//localhost:8080{$this->path}");
```

Comment:
I don't know if you can really modify that on laravel-mix.. 🤔 I don't see it in docs.
@JeffreyWay ?